app/schemas/models/level.js

```diff
@@ -625,6 +626,26 @@ _.extend(LevelSchema.properties, {
   product: _.extend(c.singleProduct, { inEditor: true })
 })
 
+LevelSchema.definitions = {
+  classroomSub: {
+    title: 'Classroom Subscription',
+    description: 'Whether this level is available in classroom content for subscribers only.',
+    type: 'object',
+    properties: {
+      default: {
```

Comment:
do we have `default` as keyword? Like if we wanted by default to have value of free, will the name cause issue?

Reply:
Ya, can we use a different name then to avoid the confusion? Maybe something like `base` instead?